README.md

```diff
@@ -309,6 +310,12 @@ If you use the `HttpBodyStream` the whole transfer will be [chunked encoded](htt
 
 Check out the `examples` folder how your computation could look like.
 
+### HTTPS Server
+
+The HTTP server can be set to an HTTPS server by using the [SecureServer from the socket package of ReactPHP](https://github.com/reactphp/socket#secureserver).
+
+Check out the `examples` folder how your HTTPS server could look like.
```

Comment:
Does it make sense to add some short snippet here that highlights what is really needed here and/or link to the appropriate example instead of the whole folder?